neo4j/_async/io/_pool.py

```diff
@@ -445,7 +459,6 @@ def __init__(self, opener, pool_config, workspace_config, address):
         """
         super().__init__(opener, pool_config, workspace_config)
         # Each database have a routing table, the default database is a special case.
-        log.debug("[#0000]  C: <NEO4J POOL> routing address %r", address)
```

Comment:
Is this information covered by other logs?

Reply:
Yes, https://github.com/neo4j/neo4j-python-driver/pull/849/files#diff-13fed080a378db561eb9e27594af933152c74fe8d8ac652fcdc1db62653c74baR395 and https://github.com/neo4j/neo4j-python-driver/pull/849/files#diff-13fed080a378db561eb9e27594af933152c74fe8d8ac652fcdc1db62653c74baR411 replace this one.